### ISSUE TITLE
Update picture element to allow aria-hidden

### ISSUE THREAD

**Issue by scottaohara** [validator/validator#1239]
While we do not think it's a good idea to allow additional roles / aria-* attributes on the picture element, since it is not supposed to be directly exposed to the a11y-api, there are potential use cases for setting the entire picture, and thus the graphics it represents as hidden.  While this should be done with `alt=""` on the fallback image element, allowing `aria-hidden=true` on the `picture` itself should be allowed as this will also allow authors to hide the element.

Action item:
The `aria-hidden` attribute can be specified on the `picture` element as the only allowed aria attribute.
